changelog.md

```diff
@@ -1,3 +1,44 @@
+# 1.2
+
+## Features
+* Added multi objective via Par-EGO.
+
+## Major Changes
+* Results are instantly saved by default now.
+* Determinstic behaviour is default now.
```

Comment:
Could you explain further what this means, and for which facade this applies to?